resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -148,6 +148,34 @@ func (c *APIClient) CallAPI(path string, method string,
 		}
 	}
 
+	if c.configuration.ProxyConfiguration != nil {
+
+		var proxyUrl *url.URL
+
+		if c.configuration.ProxyConfiguration.Auth != nil && c.configuration.ProxyConfiguration.Auth.UserName != "" && c.configuration.ProxyConfiguration.Auth.Password != "" {
+			proxyUrl = &url.URL {
+      		Scheme: c.configuration.ProxyConfiguration.Protocol,
+     	    User:   url.UserPassword(c.configuration.ProxyConfiguration.Auth.UserName, 
+	  							c.configuration.ProxyConfiguration.Auth.Password),
+      		Host:   c.configuration.ProxyConfiguration.Host + ":" + c.configuration.ProxyConfiguration.Port,
+    		}
+		} else {
+
+			urlString := c.configuration.ProxyConfiguration.Protocol + "://" +
+						c.configuration.ProxyConfiguration.Host  + ":" +
+						c.configuration.ProxyConfiguration.Port
+			fmt.Println(urlString)
```

Comment:
We should remove the fmt.Println.  If you need to log them use log.Printf

Reply:
Done.. I removed all the fmts. somehow this escaped my eyes.